esmvaltool/diag_scripts/shared/plot/_plot.py

```diff
@@ -203,20 +407,20 @@ def multi_dataset_scatterplot(x_data, y_data, datasets, filepath, **kwargs):
             style['facecolor']
 
         # Plot
-        axes.plot(
-            x_data[idx],
-            y_data[idx],
-            markeredgecolor=style['color'],
-            markerfacecolor=facecolor,
-            marker=style['mark'],
-            **(kwargs.get('plot_kwargs', empty_dict)[idx]))
+        axes.plot(x_data[idx],
+                  y_data[idx],
+                  markeredgecolor=style['color'],
+                  markerfacecolor=facecolor,
+                  marker=style['mark'],
+                  **(kwargs.get('plot_kwargs', empty_dict)[idx]))
```

Comment:
You if add a trailing comma, `yapf` will format this kind of thing nicer

Reply:
See above; these changes were made by an autoformatter (it actually removed the trailing comma at some point I think), but I will run `pre-commit` again.